esmvaltool/cmor/tables/custom/CMOR_lwp.dat

```diff
@@ -6,11 +6,11 @@ modeling_realm:    atmos
 !----------------------------------
 ! Variable attributes:
 !----------------------------------
-standard_name:     atmosphere_cloud_liquid_water_content
+standard_name:     
 units:             kg m-2
 cell_methods:      time: mean
 cell_measures:     area: areacella
-long_name:         liquid water path
+long_name:         Condensed Water Path
```

Comment:
Condensed water path is usually liquid + ice (i.e. clwvi), which is not the case here. I would therefore put "Liquid Water Path" as long_name.

Reply:
Done.